scale/scale_facade.go

```diff
@@ -212,6 +212,8 @@ func (f *ScaleFacade) calculateMaxScaleReplicas(item *corev1.ConfigMap, currentR
 
 		log.V(1).Info("metric values and replica proposal", "replicas", replicaProposal, "metrics", metricValues)
 
+		log.V(1).Info("metric values and replica proposal", "replicas", replicaProposal, "metrics", metricValues)
```

Comment:
This seems duplicate?

Reply:
@yaksharma can we remove duplicate line?

Reply:
Done.